pydantic_odm/encoders/mongodb.py

```diff
@@ -18,14 +20,10 @@ def __call__(self, data: "DictStrAny") -> "DictStrAny":
         raise NotImplementedError()
 
 
-def _convert_enums(
-    data: Union["DictStrAny", List[Any]]
+def _recursive_iterator(
+    data: Union["DictStrAny", List[Any]], transform_func
```

Comment:
You need to create a type for the transform functions and specify here (for mypy)

Reply:
Typings added in: dada6db94ba58c708033c7180b5e58023323284e

Hmm, I am having issues with flake8 conflicting with what black does. When I run `make lint` I get:

```
$ make lint
flake8 pydantic_odm/ tests/
tests/encoders/mongodb.py:95:52: E231 missing whitespace after ','
tests/encoders/mongodb.py:96:55: E231 missing whitespace after ','
tests/encoders/mongodb.py:100:62: E231 missing whitespace after ','
tests/encoders/mongodb.py:101:65: E231 missing whitespace after ','
tests/encoders/mongodb.py:107:72: E231 missing whitespace after ','
tests/encoders/mongodb.py:111:75: E231 missing whitespace after ','
make: *** [lint] Error 1
```
flake8 immediately fails over some whitespace that black does not fix for me (`make format` should fix that right?).
Then ofcourse the mypy, isort and black checks are not executed. Do you have this issue as well? If not, maybe one of us has some local configuration hanging somewhere.

Reply:
Googled and found https://github.com/psf/black/issues/1289 which points to https://github.com/psf/black/issues/1288 for the fix. Basically this is some kind of magic comma happening here. Apparantly if I remove it manually, black wont make it come back.

Reply:
Resolved linting issues with: 135fb9cce1fe129ae7974d155e2a099e671a10f0